### DISCUSSION TITLE
节点名称过长时怎么配置才能让节点名称显示省略号，鼠标放上去才展示完全呢？

### DISCUSSION THREAD

**Discussion by huangdonghai123** [antvis/X6#3917] [Q&A]
节点名称过长时怎么配置才能让节点名称显示省略号，鼠标放上去才展示完全呢？

**Comment by RainBondsongyg** [comment]
textWrap: {
              width: '100%', // 宽度为参照元素宽度
              height: '50%', // 高度为参照元素高度的一半
              ellipsis: true, // 文本超出显示范围时，自动添加省略号
            }
我是配置以上的参数，然后鼠标移动上去把高度改成100%就可以，不知道是否对你有帮助。
